docs/README.md

```diff
@@ -8,7 +8,7 @@
 
 - `Stanford.NLP.NET` is built on top of [IKVM.NET](https://github.com/ikvm-revived/ikvm) (`Java` VM that runs on top of `.NET` VM).
 - It supports full `.NET` framework and `.NET Core`.
-- You should always start from [CoreNLP](corenlp) master package that provide full range of features (other packages are exist for historical/compatibility reasons)
+- You should always start from [CoreNLP](https://www.nuget.org/packages/Stanford.NLP.CoreNLP/) master package that provide full range of features (other packages are exist for historical/compatibility reasons)
```

Comment:
@sergey-tihon I'm not sure what this was supposed to point to. The link was broken and this was my best guess.